src/cascade/saver/generate_draws.py

```diff
@@ -0,0 +1,77 @@
+"""
+Generate draws by getting data from the predict table and adding it to data
+from the avgint table.
+"""
+
+import logging
+
+import pandas as pd
+
+CODELOG = logging.getLogger(__name__)
+MATHLOG = logging.getLogger(__name__)
+
+INTEGRAND_ID_TO_MEASURE_ID_DF = pd.DataFrame([
```

Comment:
This belongs with the db code. Converting back and forth between our internal representation (integrand_id) and the external representation (measure_id) is the sort of normalization which should happen with the database code so that all the logic directly related to the external dbs lives in one place.